frontend/src/_helpers/appUtils.js

```diff
@@ -901,6 +901,14 @@ export const getSvgIcon = (key, height = 50, width = 50) => {
   return <Icon style={{ height, width }} />;
 };
 
+export const assetPath = (path) => {
+  if (isEmpty(__webpack_public_path__)) return path;
+
+  if (__webpack_public_path__.at[-1] !== '/') return `${__webpack_public_path__}${path}`;
```

Comment:
```suggestion
  if (__webpack_public_path__.at(-1) !== '/') return `${__webpack_public_path__}${path}`;
```
@akshaysasidrn , this fixes the problems on my local that I reported with screenshots earlier